src/js/stores/GroupsStore.js

```diff
@@ -0,0 +1,36 @@
+var EventEmitter = require("events").EventEmitter;
+
+var AppDispatcher = require("../AppDispatcher");
+var GroupsEvents = require("../events/GroupsEvents");
+
+var Util = require("../helpers/Util");
+
+var GroupsStore = Util.extendObject(EventEmitter.prototype, {});
+
+AppDispatcher.register(function (action) {
+  switch (action.actionType) {
+    case GroupsEvents.SCALE_SUCCESS:
+      GroupsStore.emit(GroupsEvents.SCALE_SUCCESS);
```

Comment:
OK but it's kind of hard to understand why we need it for the `DELETE_SUCCESS` case in this PR then... :) 
I guess I'm more in favour of predictability if I can't see how stuff is going to be used :wink: